Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM pangeo/base-image:2020.06.03
+FROM pangeo/base-image:2021.07.17
```

Comment:
I did not go for the latest hash (published 3 days) but went for this tagged one instead. (See https://hub.docker.com/r/pangeo/pangeo-notebook/tags?page=1&ordering=last_updated). We should be OK.